ssh/id_busybee

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Let's not support ssh into the box FOR user: see the comment at the bottom. If we were doing this, I'd ask the make to generate the pair of keys on image build, rather than keep it in public.